.CI/trigger_travis_conda-forge.github.io.sh

```diff
@@ -0,0 +1,39 @@
+"""
+Trigger the conda-forge.github.io Travis job to restart.
+
+"""
+import requests
+import six
```

Comment:
Ah :hankey: - not enough sleep clearly. Thanks @jakirkham. I'll submit a third PR :disappointed: